integrations/docker/images/chip-build-android/version

```diff
@@ -0,0 +1 @@
+0.2.16
```

Comment:
I think this is out of date now, since the ThreadStackManagerImpl went in for Linux, we're at 0.2.18?

Reply:
done